Makefile

```diff
@@ -8,12 +8,25 @@ doc:
 	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
 	@echo
 
+doc3:
+	python3 setup.py build_sphinx
+	@echo
+	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
+	@echo
+
 test:
 	-find coverage/ -mindepth 1 -delete
 	python $$(which nosetests) $${TESTS}
 
+test3:
+	-find coverage/ -mindepth 1 -delete
+	python3 $$(which nosetests3) $${TESTS}
+
 clean:
 	find . -name '*.py[co]' -delete
 
 dist: test
 	python setup.py sdist
+
+dist3: test3
+	python3 setup.py sdist
```

Comment:
You're right.  I've been using python3 in my invocations for building the package using stdeb, and blindly copied it in for creating the source distribution.